azure-mgmt-storage/src/main/java/com/azure/management/storage/implementation/ImmutabilityPolicyImpl.java

```diff
@@ -116,20 +124,17 @@ public ImmutabilityPolicyImpl withExistingContainer(String resourceGroupName, St
 
     @Override
     public ImmutabilityPolicyImpl withIfMatch(String ifMatch) {
```

Comment:
Seems better to use unified call e.g. `withETagCheck`?

BTW does it have the implicit version `withETagCheck()`?

Reply:
For storage, it only has `withIfMatch(ifMatch)`. And it's not optional. It means users have to call `withIfMatch(null)` if they want to select the default way.

Reply:
What I mean is to set the default implicit, as the `withETagCheck()` which sets to "*" or "current resource etag" based on whether on create/update.